rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -82,19 +110,22 @@ extern s32 sysutil_send_system_cmd(u64 status, u64 param)
 			}
 		}
 
-		for (sysutil_cb_manager::registered_cb cb : cbm->callbacks)
+		for (auto& cb : cbm->callbacks)
 		{
-			if (cb.first)
+			cb.atomic_op([&](auto& cb)
```

Comment:
atomic load is much cheaper than atomic cmpxchg + atomic load. atomic cmpxchg should only be used writing.